### ISSUE TITLE
[Build Issue] gzguts missing unistd.h?

### ISSUE THREAD

**Issue by OdinVex** [TTK-qmmp/qmmp-psf#2]
I've needed to include `unistd.h` in `./libpsf/zlib/gzguts.h` recently, otherwise a number of functions aren't defined (for Linux, Mac). There's code in `./libpsf/zlib/zconf.h` that uses it `unistd.h` though.

**Comment by Greedysky** [comment]
em... I have no such problem  on Ubuntu Linux, maybe you have to send an issue to https://github.com/madler/zlib.

**Comment by OdinVex** [comment]
It may be something specific to Manjaro/Arch-based. I maintain the AUR `qmmp-pfs-git` package. It compiled before when using commit 06f2d36 (long ago, though) but I updated the version to 12e0d3e and now it appeared, so maybe it's distro-related or yes, perhaps zlib. Which version of zlib are you using?

Edit: Found an Issue on zlib, may be related, will follow up when I have some free time: https://github.com/madler/zlib/issues/923
